tests/core/test-core.c

```diff
@@ -30,8 +30,52 @@ static void test_brick_core_simple_lifecycle(void)
 {
 	struct pg_error *error = NULL;
 	struct pg_brick *brick;
-	struct pg_brick_config *config = pg_brick_config_new("mybrick", 2, 2,
-							     PG_MULTIPOLE);
+	struct pg_brick_config *config = pg_brick_config_new("mybrick", 2, 2, PG_MULTIPOLE);
+	struct pg_brick_config *config1 = pg_brick_config_new("mybrick", 2, 2, PG_MONOPOLE);
+	struct pg_brick_config *config2 = pg_brick_config_new("mybrick", UINT16_MAX, UINT16_MAX, PG_MULTIPOLE);
```

Comment:
I think config need to be free, it doesn't seems you free the configs

Reply:
done

Reply:
done